include/seqan3/io/stream/parse_condition_detail.hpp

```diff
@@ -300,7 +300,6 @@ class parse_condition_base
      */
     //!\brief Returns the message representing this condition as std::string.
     std::string message() const
-        requires parse_condition_concept<derived_t>
```

Comment:
maybe include a `static_assert` within the function body to still ensure the requirement